sdk/tables/azure-data-tables/samples/tables_getting_started.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <azure/data/tables.hpp>
+
+#include <cstdio>
+#include <iostream>
+#include <stdexcept>
+#include <thread>
+
+std::string GetConnectionString()
+{
+  const static std::string ConnectionString = "";
+
+  if (!ConnectionString.empty())
+  {
+    return ConnectionString;
+  }
+  const static std::string envConnectionString = std::getenv("STANDARD_STORAGE_CONNECTION_STRING");
```

Comment:
`STANDARD_STORAGE_CONNECTION_STRING` is used in Storage samples.
Can I as a customer have it defined to work with the previous Storage samples, and run this sample, and have it run successfully?

If the answer is no, please rename it to something like `DATA_TABLES_CONNECTION_STRING`

Reply:
Yes i connect to the storage accoutn, it's the same conn string that could be used by blob service for example